PicoPagesImages.php

```diff
@@ -1,12 +1,16 @@
 <?php
 /**
  * Get the images of the current page with {{ images }} in Pico CMS.
+ * Edited September 2018 by Brian Goncalves
  *
  * @author  Nicolas Liautaud <contact@nliautaud.fr>
  * @license http://opensource.org/licenses/MIT The MIT License
  * @link    http://nliautaud.fr
  * @link    http://picocms.org
  */
+
+use Symfony\Component\Yaml\Yaml;
```

Comment:
Is there any requirement to document here ?

Reply:
Actually I just found the way to call the yaml parser through the pico class—will do that instead.

Reply:
Great 👍